sv_pipeline/genome/load_data.py

```diff
@@ -38,7 +38,8 @@
                                              hl.missing(hl.tstr)),
     'gnomad_svs_AF': lambda rows: rows.info.gnomAD_V2_AF,
     'pos': lambda rows: rows.locus.position,
-    'filters': lambda rows: hl.array(rows.filters.filter(lambda x: x != 'PASS')),
+    'filters': lambda rows: hl.array(rows.filters.filter(lambda x: (x != 'PASS') & (x != 'BOTHSIDES_SUPPORT'))),
+    'bothsides_support_filter': lambda rows: rows.filters.any(lambda x: x == 'BOTHSIDES_SUPPORT'),
```

Comment:
so its not actually a filter (which is part of why its so confusing that it was in the filters field to begin with) so just name it `bothsides_support`